### DISCUSSION TITLE
Quarkus OpenTelemetry and MongoDB

### DISCUSSION THREAD

**Discussion by memorais** [quarkusio/quarkus#24455] [Q&A]
Hi there,

I was able to configure OpenTelemetry in my Quarkus application and send spans to Jaeger. The problem is I can't see MongoDB spans and I was wondering if there is some specific configuration to make this work properly or if this remain unsupported right now. When using OpenTracing library, it's possible to see spans from MongoDB interactions. Can someone clarify this please?

Thanks.

[a comment by an automated account is not shown]

**Reply by radcortez** [reply]
We don't have any specific OTel Mongo support at this moment. We are slowly building the OTel support. The OTel library may or may not work, but we don't have integrated so we don't know what works or not.

**Reply by memorais** [reply]
Hi @radcortez, I got it. 

Thanks for your reply.

**Reply by missaeldenadai-openco** [reply]
Hi @radcortez.

Any update on that?

**Reply by radcortez** [reply]
I created an issue some time ago: https://github.com/quarkusio/quarkus/issues/28473

But no progress so far. Maybe you could help? :)
